breadStore/main.js

```diff
@@ -57,4 +57,15 @@ modalCloseBtn.forEach((closeBtn) => {
   closeBtn.addEventListener('click',closeModal) });
 
 
+// 카테고리 태그 만들기
+function addCategoryTag(addId) {
+  let resultList = ``
+  const categorySection = document.querySelector("#section__category");
+  resultList = `
+  <div id="tag-${addId}">${addId}<button class="typeTag" id="tag-btn-${addId}" type="button"> X</button></div>
+  `;
```

Comment:
```suggestion

 const categorySection = document.querySelector("#section__category");
 const resultList = `
  <div id="tag-${addId}">${addId}<button class="typeTag" id="tag-btn-${addId}" type="button"> X</button></div>
  `;
```
로하면 let안써도 데지않나?!! 왜냠 js도 그렇고 리액트도 그렇고 let은 변경이 쉬워서 의도대로 작동하지 않을수도 있어서 최대한 지양하는게 조하!